README.md

```diff
@@ -71,6 +71,10 @@ Apache Polaris is built using Gradle with Java 21+ and Docker 27+.
 - `./gradlew run` - To run the Polaris server locally; the server is reachable at 
```

Comment:
Could you briefly mention that this run method uses preset `root:secret` credentials?